### DISCUSSION TITLE
Updated website

### DISCUSSION THREAD

**Discussion by kjk** [sumatrapdfreader/sumatrapdf#2549] [Announcements]
I've updated SumatraPDF website: https://www.sumatrapdfreader.org/

Not drastically different but hopefully a bit nicer.

Also cleaned up manual a bit.

For a limited time: the old website looked like https://sumatra-website-preview.onrender.com/free-pdf-reader

**Comment by quyleanh** [comment]
Cool.
It will be nicer if you make the "Everyone loves SumatraPDF" a bit more responsive.

**Reply by kjk** [reply]
done (will be 2 columns for screens under 1200)

**Reply by quyleanh** [reply]
It's better now.
One more thing is the font size of each tab is inconsistent in mobile.

**Reply by kjk** [reply]
I don't quite follow, a screenshot would be useful.
Do you mean "font size on the same page" or "font size across pages".
And what is "mobile" in this context? iOS? Android? Simulating small screen with Chrome Dev Tools?

**Reply by kjk** [reply]
I see, on Android the font size on main page is much smaller than on download / manual. And on those pages it's too big.

**Reply by quyleanh** [reply]
Yep. That is the issue.

**Reply by kjk** [reply]
Ok, I've improved mobile look as well.

**Reply by quyleanh** [reply]
Great. Thank you.

**Comment by kennytv** [comment]
Definitely looks more inviting now!

**Comment by Alexeilive** [comment]
It's nice and clearer, well done, thanks,
Alexei
